.github/pull_request_template.md

```diff
@@ -2,6 +2,11 @@
 "/invoices/:id" to show a form same as "/invoices-insert" that shows the fields filled with the actual data in order to update or cancel.
 
 ## Proposed changes
-add update endpoint and logic. make a view.
+- add update endpoint and logic. make a view.
+- need to spec what the amount field in the form.
+- submitting the form should return "/invoices"
 
 ## PR status, roadblocks, etc
+ - :id and :amount pass as form-params in "/invoices-update" and can be updated in the form yet id shouldn't be update as it is a SERIAL in the DB.
+ 
```

Comment:
the reason the :id is in the form is to carry it over to the "UPDATE users SET amount ? WHERE id ,,,"
Need a different way to carry :id not as form-param

Reply:
Gotchu. The proper way would be to refactor the `"/invoices-update"` route into `"/invoices-update/:id"`. Then you get the id from :path-params, and the email name from :form-params.

Please do give it a try